src/equations/shallow_water_exner_1d.jl

```diff
@@ -0,0 +1,618 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+#! format: noindent
+
+# Abstract type for the different bottom friction models
```

Comment:
Yes, since most test cases only consider Manning friction I removed the Darcy-type friction for now. 
However, I kept the friction type so we can easily add other friction formulations if needed.